providers/providertest/provider.go

```diff
@@ -111,7 +111,7 @@ func (c *Config) Validate() error {
 func DefaultProviderTestConfig() Config {
 	return Config{
 		TestDuration:   1 * time.Minute,
-		PollInterval:   5 * time.Second,
+		PollInterval:   20 * time.Second,
```

Comment:
Any justification for changing the poll interval?

Reply:
reverted this, was just accidentally committed as I was experimenting